src/PMGDQueryHandler.cc

```diff
@@ -1021,6 +1021,38 @@ void insert_into_queue(std::list<AutoDeleteNode *> *queue,
   }
 }
 
+void PMGDQueryHandler::print_node_idx_stats(char *tag_name, char *prop_id) {
+
+  try {
+    Transaction tx(*_db, Transaction::ReadWrite);
+    struct Graph::IndexStats istats =
+        _db->get_index_stats(Graph::NodeIndex, tag_name, prop_id);
+    printf("Index Stats for : %s %s\n", tag_name, prop_id);
+    printf("Total Entries: %lu\n", istats.total_unique_entries);
+    printf("Unique Entry Size: %lu\n", istats.unique_entry_size);
+    printf("Total Elements: %lu\n", istats.total_elements);
+    printf("Total Size Bytes: %lu\n", istats.total_size_bytes);
+    printf("Health Factor: %lu\n", istats.health_factor);
+    tx.commit();
+  } catch (PMGD::Exception e) {
+    print_exception(e);
+  }
+}
+
```

Comment:
May be able to extend existing Unit tests to get coverage here. Need to look into.